textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/StringListField.java

```diff
@@ -0,0 +1,36 @@
+package edu.uci.ics.textdb.common.field;
+
+import edu.uci.ics.textdb.api.common.IField;
+
+import java.util.List;
+
+/**
+ * Created by kishorenarendran on 25/04/16.
+ */
+public class StringListField implements IField {
```

Comment:
@kishore-narendran ..This class won't be required once you use the generic ListField.